factory.go

```diff
@@ -23,7 +22,7 @@ type registryEntry struct {
 	Info ComponentInfo
 }
 
-// FactoryConfig sets up properties of a Factory
+// FactoryConfig sets up properties of a Factory.
 type FactoryConfig struct {
 	RegistryCapacity uint
```

Comment:
One more capacity config is here

Reply:
After a bit of consideration, I dropped the `FactoryConfig` altogether. If anyone were to use it in the past, removing `RegistryCapacity` would break the API anyway. On the other hand, keeping an empty struct "just in case" did not feel right.